helper/scss.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 module.exports = function(gulp, plugins, config, name, file) { // eslint-disable-line func-names
-  const theme       = config.themes[name],
-        srcBase     = config.projectPath + 'var/view_preprocessed/frontools' + theme.dest.replace('pub/static', ''),
-        stylesDir   = theme.stylesDir ? theme.stylesDir : 'styles',
-        disableMaps = plugins.util.env.disableMaps || false,
-        production  = plugins.util.env.prod || false,
-        postcss     = [];
+  const theme         = config.themes[name],
+        srcBase       = config.projectPath + 'var/view_preprocessed/frontools' + theme.dest.replace('pub/static', ''),
+        stylesDir     = theme.stylesDir ? theme.stylesDir : 'styles',
+        disableMaps   = plugins.util.env.disableMaps || false,
+        production    = plugins.util.env.prod || false,
+        postcss       = [],
+        disableSuffix = theme.disableSuffix || false;
```

Comment:
It's consistent with the rest of the code, but FYI the `|| false` is unnecessary because it'll be `undefined` if you haven't set it.